lib/core/include/irods/rodsErrorTable.h

```diff
@@ -161,9 +161,13 @@ NEW_ERROR(SYS_DIR_IN_VAULT_NOT_EMPTY,                  -82000)
 NEW_ERROR(SYS_OPR_FLAG_NOT_SUPPORT,                    -83000)
 NEW_ERROR(SYS_TAR_APPEND_ERR,                          -84000)
 NEW_ERROR(SYS_INVALID_PROTOCOL_TYPE,                   -85000)
+
+// The following three codes are EOL since iRODS v5.0.0
 NEW_ERROR(SYS_UDP_CONNECT_ERR,                         -86000)
 NEW_ERROR(SYS_UDP_TRANSFER_ERR,                        -89000)
 NEW_ERROR(SYS_UDP_NO_SUPPORT_ERR,                      -90000)
+// END EOL Block
+
```

Comment:
Use the same comment style as this so it starts to become convention and greppable.

https://github.com/irods/irods/blob/d8117e2191186465198f67b116ac0d03045f5050/lib/core/include/irods/rodsErrorTable.h#L139

Reply:
Done.